scanpy/external/_tools/_palantir.py

```diff
@@ -0,0 +1,297 @@
+"""Run Diffusion maps using the adaptive anisotropic kernel
+"""
+
+from scanpy import logging as logg
+
+
+def palantir(adata):
+    """
+    Run Diffusion maps using the adaptive anisotropic kernel [Setty18]_.
+
+    Palantir is an algorithm to align cells along differentiation trajectories.
+    Palantir models differentiation as a stochastic process where stem cells
+    differentiate to terminally differentiated cells by a series of steps through
+    a low dimensional phenotypic manifold. Palantir effectively captures the
+    continuity in cell states and the stochasticity in cell fate determination.
+    Palantir has been designed to work with multidimensional single cell data
+    from diverse technologies such as Mass cytometry and single cell RNA-seq.
+
+    .. note::
+        More information and bug reports `here <https://github.com/dpeerlab/Palantir>`__.
+
+    Parameters
+    ----------
+    adata: :class:`~anndata.AnnData`
```

Comment:
Sorry, but there needs to be a white space between parameters and colon: `adata : ` instead of `adata: `. :) Otherwise the docs will break.

Reply:
Also you document some parameters but the function only has one.

Reply:
Thank you @falexwolf and @flying-sheep !
I fixed the docs.

note: how can I check if the docs break? I usually run make html in docs to inspect the rendering, and I didn't get any flags

Reply:
Using `make html` is fine. Just inspect the resulting pages and you'll see that the layout is broken if something goes wrong.